docs/source/building_an_experiment.rst

```diff
@@ -2,49 +2,306 @@
+    12 MHz. Note that this is tuned so will be set to a quantized value, and 
+    the value can be accessed by the user at this attribute after the experiment 
+    begins.
+
+decimation_scheme
+    The decimation scheme for the experiment, provided by an instance of the 
+    class DecimationScheme. There is a default scheme specifically set for the 
+    default rates and centre frequencies above. 
+
+comment_string
+    A comment string describing the experiment. It is highly encouraged to
+    provide some description of the experiment for the output data files. The 
+    default is '', or an empty string. 
+
+Below is an example of properly inheriting the prototype class and defining your own experiment::
 
     class MyClass(ExperimentPrototype):
 
         def __init__(self):
             cpid = 123123  # this must be a unique id for your control program.
-            super(MyClass, self).__init__(cpid)
+            super(MyClass, self).__init__(cpid, comment_string='My clever experiment.')
 
 The experiment handler will create an instance of your experiment when your experiment is scheduled to start running. Your class is a child class of ExperimentPrototype and because of this, the parent class needs to be instantiated when the experiment is instantiated. This is important because the experiment_handler will build the scans required by your class in a way that is easily readable and iterable by the radar control program. This is done by methods that are set up in the ExperimentPrototype parent class.
 
 The next step is to add slices to your experiment. An experiment is defined by the slices in the class, and how the slices interface.
 
+Experiment Slices
+-----------------
 
-TODO
+Experiment slices are any individual component of an experiment. An experiment 
+slice represents one defined pulse sequence at a single frequency and processed
+given certain parameters. A slice is defined using a dictionary and the 
+necessary slice keys.
+
+Slice Keys
+----------
+
+These are the keys that are set by the user when initializing a slice. Some 
+are required, some can be defaulted, and some are set by the experiment 
+and are read-only.
+
+**Slice Keys Required by the User**
+
+pulse_sequence *required*
+    The pulse sequence timing, given in quantities of tau_spacing, for example
+    normalscan = [0, 14, 22, 24, 27, 31, 42, 43]. 
+
+tau_spacing *required*
+    multi-pulse increment in us, Defines minimum space between pulses.
+
+pulse_len *required*
+    length of pulse in us. Range gate size is also determined by this.
+
+num_ranges *required*
+    Number of range gates.
+
+first_range *required*
+    first range gate, in km
+
+intt *required or intn required*
+    duration of an integration, in ms. (maximum)
+
+intn *required or intt required*
+    number of averages to make a single integration, if intt = None.
+
+beam_angle *required*
+    list of beam directions, in degrees off azimuth. Positive is E of N. Array
```

Comment:
This could be misleading. Our main antenna array has 16 elements, but this beam_angle array _also_ has 16 elements, and therefore 16 beams!
```suggestion
    list of beam directions, in degrees off azimuth. Positive is E of N. The beam_angle array
```